src/controllers/pages/items.js

```diff
@@ -0,0 +1,12 @@
+import { getAllModels } from '../../lib/api-client';
+import { Dispatcher } from 'consus-core/flux';
+import { hashHistory } from 'react-router';
+
+export default class ItemController {
+    static newItem() {
+        return getAllModels().then(models => {
```

Comment:
I think this call gets all the models and takes you to the page to create an item. I can make that more explicit.